src/09-workorders/work-orders.apib

```diff
@@ -94,6 +97,9 @@ Get details for a single work order.
             + department (object, nullable)
                 + type: `department` (string)
                 + id: `5e90eb0a-b502-4344-aa0f-3b8525af6186` (string)
+            + document_template (object)
```

Comment:
B:
```suggestion
            + document_template (object, nullable)
```

Reply:
So we don't return the default one, like it's said in the description?

Reply:
Like tiago said, we do know which template will be used. Why not return the default one?
(you mentioned that this is not the case for the department. Should we go for consistency or complete data?)

Reply:
We actually do not do this for the department either. Why would we for the document template?

It only matters when we are finalizing. When it's not set on the WO, the default is being used.

Reply:
Made it nullable 😉

Reply:
but a (draft) workorder always has a department and a template, no?
so why would you hide that from the API users?

Reply:
This can give two slightly weird scenarios though:

1. When upon drafting the default template is assigned in the background, and that default template is later changed to a different one, the WorkOrders that were not yet finalized will all be created with the old template.
1. When we return the current default template for every WO without one explicitly set, changing the default template will make workOrders.info suddenly return a different template. This can be confusing for an integrator that a value changed. Although this might be the best scenario in this case.

But I would also keep it as is and see if we need to change this later on.